daemon/labels.go

```diff
@@ -232,16 +233,34 @@ func NewGetIdentityHandler(d *Daemon) GetIdentityHandler {
 }
 
 func (h *getIdentity) Handle(params GetIdentityParams) middleware.Responder {
-	lbls := labels.NewLabelsFromModel(params.Labels)
-	if id, err := LookupIdentityBySHA256(lbls.SHA256Sum()); err != nil {
-		if apierr, ok := err.(*apierror.APIError); ok {
-			return apierr
+	log.Debugf("GET /identity request: %+v", params)
```

Comment:
debug log left here by mistake?

Reply:
not by mistake. I followed the style in endpoints.go. Moreover, this helped me while debugging.